frontend/apps/reader/modules/readerlink.lua

```diff
@@ -67,6 +67,7 @@ local ReaderLink = InputContainer:extend{
     location_stack = nil, -- table, per-instance
     forward_location_stack = nil, -- table, per-instance
     _external_link_buttons = nil,
+    handleableSchemes = {"http", "https"},
```

Comment:
`supportedLinkProtocols`/schemes? Or `handledSchemes` if you like the word handle? `handleable` seems a bit ugly to me. ^_^